app/components/formatted_date/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {renderWithIntl} from '@test/intl-test-helper';
+
+import locales from '../../i18n/languages';
+
+import FormattedDate, {type FormattedDateFormat} from './index';
+
+const DATE = new Date('2024-10-26T10:01:04.653Z');
+const FORMATS = [
+    undefined,
+    {weekday: 'long'},
+    {dateStyle: 'medium'},
+    {month: 'short', day: 'numeric'},
+    {
+        month: 'short',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+    },
+    {
+        year: 'numeric',
+        month: 'short',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+    },
+] satisfies Array<FormattedDateFormat | undefined>;
+
+const TEST_MATRIX = Object.keys(locales).flatMap((locale) => FORMATS.map((format) => ({format, locale})));
+
+describe('<FormattedDate/>', () => {
+    it.each(TEST_MATRIX)('should match snapshot', ({format, locale}) => {
```

Comment:
Can you make so tests names are more meaningful? You can take a look on how to use the name templates here:

https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Sure !